cmd/mattermost/commands/user.go

```diff
@@ -368,6 +379,52 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(command *cobra.Command, args []string, a *app.App) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	users := getUsersFromUserArgs(a, args)
```

Comment:
Probably we can move this out of the function, allowing that all arguments handling is done at command handler level, and then pass to this function a `[]*model.User`.

Reply:
The plan is to implement user-to-bot first as in this PR. I am having a follow up that does bot-to-user. I want to completely separate 2 workflows, which is why everything falls in `usersToBots(...)`